src/App.js

```diff
@@ -107,25 +125,50 @@ function App() {
     });
   }, [])
 
+  const printScore = (score) => {
+    const txtColor = score > 0.55 ? "text-black" : "text-red-400";
+    score = String(score).slice(0, 6);
+
+    return (
+      <h2 className={txtColor}>{score}</h2>
+    )
+  }
+
   return (
     <div className="App">
-      <input
-        id="standard-read-only-input"
-        type="text"
-        label="Type your sentences here"
-        onChange={(e) => setText(e.target.value)}
-        value={testText}
-        rows={4}
-        variant="outlined"
-      />
-      <br />
-      <br />
-      {testText !== "" ?
-        <button style={{ width: "20vh", height: "5vh" }} variant="outlined" onClick={() => getSentimentScore(testText)}>Calculate</button>
-        : <></>}
-      {testScore && (
-        <h2>{testScore}</h2>
-      )}
+      <div className="md:w-3/6 h-80 overflow-auto md:mx-auto mx-2 mt-2 p-2 bg-gray-200">
```

Comment:
To set an element centered on screen, use `className="container mx-auto"` or `className="max-w-md"` (or another max-width style)
[Reference to tailwindcss max-width classes](https://tailwindcss.com/docs/max-width